src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.AppHost/Properties/launchSettings.json

```diff
@@ -21,7 +22,8 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
         "DOTNET_ENVIRONMENT": "Development",
-        "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:18000"
+        "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:21000",
```

Comment:
I think this breaks dapr

Reply:
Oh? Why's that? Reserved port range?

Reply:
I'm not sure they handle https to the otlp endpoint. Try out the sample and see if telemetry still works.

Reply:
Seems to be working fine 
![image](https://github.com/dotnet/aspire/assets/249088/8cf8ebb6-19c6-417b-9161-e072bc1fe031)

Reply:
Change I made locally to test this is at https://github.com/dotnet/aspire-samples/commit/f97b0e7bb421452de4f4042e01d4bb24d420a440